### PR TITLE
update version { realio , planq }

### PR DESCRIPTION
**Pull request by Genz22**
- add latest binary version : realio , planq
- add endpoint for planq
- fix endpoint bluzelle

### CODE REVIEW

File: planq/chain.json [hunk 1]
```diff
@@ -33,19 +33,23 @@
   },
   "codebase": {
     "git_repo": "https://github.com/planq-network/planq",
-    "recommended_version": "v1.0.3",
+    "recommended_version": "v1.0.7",
```
Comment (JeremyParish69) [inline planq/chain.json:36]: For planq, if v1.0.7 is incompatible with v1.0.0-v1.0.3 (consensus-breaking), then can we please add v1.0.7 as a new object under `versions` as well?

File: realio/chain.json [hunk 2]
```diff
@@ -97,12 +97,26 @@
           "version": "0.34"
             },
           "ibc_go_version": "6.1.1",
+          "next_version_name": "v0.8.3"
+        },
+        {
+          "name": "v0.8.3",
+          "recommended_version": "v0.8.3",
+          "compatible_versions": [
+            "v0.8.2","v0.8.3"
```
Comment (JeremyParish69) [inline realio/chain.json:106]: if v0.8.2 is compatible with v0.8.3, then we simply update the existing v0.8.2 version object (but it looks like you've added a new object, which could be confusing)

Comment (Genz22) [reply realio/chain.json:106]: yeah v0.8.2 is compatible with v0.8.3

File: planq/chain.json (hunk 1 above)
Comment (Genz22) [reply planq/chain.json:36]: yes please

**Comment by Genz22** [comment]
check pls

File: realio/chain.json (hunk 2 above)
Comment (JeremyParish69) [reply realio/chain.json:106]: Sorry, this was never resolved. There is still a separate version object for v0.8.2 that should be removed
image

Comment (Genz22) [reply realio/chain.json:92]: https://github.com/cosmos/chain-registry/pull/2663/commits/eadf6aeb331e028e299dd749c39da976ba4204c7

File: planq/chain.json [hunk 3]
```diff
@@ -65,6 +70,20 @@
           "darwin/arm64": "https://github.com/planq-network/planq/releases/download/v1.0.3/planq_1.0.3_Darwin_arm64.tar.gz",
           "windows/amd64": "https://github.com/planq-network/planq/releases/download/v1.0.3/planq_1.0.3_Windows_x86_64.zip"
         }
+      },
+      {
+        "name": "v1.0.7",
+        "recommended_version": "v1.0.7",
+        "compatible_versions": [
+          "v1.0.7"
+        ],
```
Comment (JeremyParish69) [inline planq/chain.json:79]: can we add the cosmos_sdk_version, consensus, and ibc_go_version into the v1.0.7 version object please?

Comment (Genz22) [reply planq/chain.json:79]: added https://github.com/cosmos/chain-registry/pull/2663/commits/2acdf9fa1aacc451e8d541b4ca1f40f1ce8a7f13

**Pull request merged**